### PR TITLE
Update KDS Theme Tokens

### LINKED ISSUE #775: Update KDS Theme Token values
When upgrading our themes for the rebrand, the new numerical token values for colors did not correspond exactly to the material specifications. This is fine in main Kolibri, but presents a challenge for upgrading custom themed Kolibri.

While we will need to make these changes in some places in Kolibri as well, the first thing will be to make the changes in KDS.

The updated mapping: 
Screenshot 2024-09-06 at 12 14 29 PM

### PR DESCRIPTION
**Pull request by AlexVelezLl**
## Description

Update KDS Theme Tokens to the latest specs in #775.

#### Issue addressed
Closes #775.

### Before/after screenshots

| Before | After |
| --- | --- |
| image | image |
| image | image |
| image | image |
## Changelog

  - **Description:** Update Theme Tokens to the latest specs to comply material design specifications.
  - **Products impact:** Updated API.
  - **Addresses:** https://github.com/learningequality/kolibri-design-system/issues/775.
  - **Components:** -.
  - **Breaking:** yes.
  - **Impacts a11y:** no.
  - **Guidance:** Please update all used `v_*` theme tokens using the mapping in https://github.com/learningequality/kolibri-design-system/issues/775.

## Steps to test

1. Look through components and see that there is no regression regarding colors
2. Check that all colors have been changed following the colors map in #775.

### At a high level, how did you implement this?

Searched for all v_* colors and replaced all of them but `gray.v_400` and `gray.v_800` which remained the same.

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical and brittle code paths are covered by unit tests
- [x] The change is described in the changelog section above

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

### CODE REVIEW

File: docs/pages/menus/index.vue [hunk 1]
```diff
@@ -91,11 +91,11 @@
         </li>
         <li>
           Option text color:
-          <DocsInternalLink code text="tokens.text" href="/colors#palette-grey-v_100" />
+          <DocsInternalLink code text="tokens.text" href="/colors#palette-grey-v_200" />
         </li>
         <li>
           Option text hover color:
-          <DocsInternalLink code text="palette.grey.v_100" href="/colors#palette-grey-v_100" />
+          <DocsInternalLink code text="palette.grey.v_200" href="/colors#palette-grey-v_100" />
```
Comment (marcellamaki) [inline docs/pages/menus/index.vue]: I think these should both be v_200? Although looking at the docs, I'm not sure that this is meaningfully displayed to the user (not your changes, just in general I'm not sure how helpful it is or how much of a difference it makes)

File: docs/pages/colors.vue [hunk 2]
```diff
@@ -203,16 +203,16 @@
...
       <DocsColorBlock name="palette.green.v_200" />
-      <DocsColorBlock name="palette.green.v_400" />
-      <DocsColorBlock name="palette.green.v_600" />
+      <DocsColorBlock name="palette.green.v_300" />
       <p style="margin-left: 60px">
         …
       </p>
-      <DocsColorBlock name="palette.green.v_1000" />
-      <DocsColorBlock name="palette.green.v_1100" />
+      <DocsColorBlock name="palette.green.v_500" />
```
Comment (marcellamaki) [inline docs/pages/colors.vue:212]: let's replace the 
```
<p style="margin-left: 60px">
    …
</p>
```

with 
`<DocsColorBlock name="palette.green.v_400" />` 

(not strictly in scope of the palette changes, but I think there isn't really a need to skip the 400 value. I think the "break" was more about the larger gap in between)

**Review by marcellamaki — COMMENTED** [review]
Thanks Alex, this looks great. I added just a couple of very small comments. I'm going to try testing this out with one of our custom themed plugins to confirm that everything is working, and then if everything looks good with that I'll approve

File: docs/pages/menus/index.vue (hunk 1 above)
Comment (AlexVelezLl) [reply docs/pages/menus/index.vue]: Whoops :sweat_smile: Fixed it!

> Although looking at the docs, I'm not sure that this is meaningfully displayed to the user

Yes, I think the components specs pages are a little outdated. I just saw that in [Appbars](https://design-system.learningequality.org/appbars/) for example, we are still using the old palette for the examples.

File: docs/pages/colors.vue (hunk 2 above)
Comment (AlexVelezLl) [reply docs/pages/colors.vue:212]: Done! I also updated the paragraph above `"and segment colors into brightness levels, named <code>v_100</code>, <code>v_200</code>, <code>v_300</code>, <code>v_400</code>, <code>v_500</code>, <code>v_600</code>:"`

**Comment by AlexVelezLl** [comment]
Thank you @marcellamaki! Just pushed the changes (:

**Review by marcellamaki — APPROVED** [review]
Looks great, thank you @AlexVelezLl. 

When testing with our "cookiecutter" theme plugin, the current version of Kolibri develop breaks, and this fixes it! (Don't mind the ugly test colors :) )

|  Before | After |
|---|---|
| Screenshot 2024-10-01 at 1 23 48 PM | Screenshot 2024-10-01 at 1 41 31 PM |

**Comment by AlexVelezLl** [comment]
Thank you @marcellamaki!

**Pull request merged**